src/server/api/controllers/users.controller.js

```diff
@@ -26,7 +26,23 @@ const getUsersById = async (id) => {
     return error.message;
   }
 };
-
+// eslint-disable-next-line
+const getUserFavorites = async (user_id) => {
+  // eslint-disable-next-line
+   if (!user_id) {
+    throw new HttpError(
+      'Bad request. user_id must be an integer and larger than 0',
+      400,
+    );
+  }
+  return knex('favorites')
+  // eslint-disable-next-line
```

Comment:
you hsould not just disable eslint.
 you can disable for 1 rule only `camelcase`
That way you don`t miss ther errors.